lib/puppet.rb

```diff
@@ -138,14 +138,14 @@ def self.run_mode
   # @api public
   # @param args [Array<String>] the command line arguments to use for initialization
   # @return [void]
-  def self.initialize_settings(args = [])
-    do_initialize_settings_for_run_mode(:user, args)
+  def self.initialize_settings(args = [], require_config: true)
```

Comment:
@jhelwig I just updated this to take named args. Though it looks like we still support Ruby 1.9.3, and wasn't this introduced in Ruby 2? It'll be interesting to see what that travis cell looks like once it's done

Reply:
Named args are a Ruby 2 thing. Did we not drop Ruby 1.9.3 support until Puppet 6?

Reply:
we're still running 1.9.3 in travis 🤷‍♀️

Reply:
Yes, still need 1.9 support in 5.x, because of JRuby 1.7.